pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java

```diff
@@ -65,18 +65,19 @@ public void testCreate() {
      */
     @Test
     public void testSave() {
-        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
+//        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
```

Comment:
@barrantesgerman `git checkput -b gzip origin/gzip`
A PR is a possible merge from a branch (`gzip` in this case).